<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentsModule</class>
 <widget class="QWidget" name="PaymentsModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payments Module</string>
  </property>
  <property name="styleSheet">
   <string notr="true">PaymentsModule {
    background-color: #f0f0f0;
    color: #333333;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 13px;
}

QLabel {
    font-size: 14px;
    color: #333333;
}

QLabel[styleSheet*="font-weight: bold"] {
    font-size: 18px;
    font-weight: bold;
}

QPushButton {
    background-color: #70B9FE;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 13px;
    min-height: 30px;
    min-width: 80px;
}

QPushButton:hover {
    background-color: #5aa9fe;
}

QPushButton:pressed {
    background-color: #4a99fe;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
}

QLineEdit:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QLineEdit:read-only {
    background-color: #e6e6e6;
    color: #666666;
}

QLineEdit[placeholderText] {
    color: #788596;
}

QComboBox {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
    min-width: 200px;
}

QComboBox:hover {
    border-color: #70B9FE;
}

QComboBox:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    selection-background-color: #70B9FE;
    selection-color: #ffffff;
    color: #333333;
}

QTextEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
    min-height: 80px;
}

QTextEdit:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QTableView {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    gridline-color: #e6e6e6;
    font-size: 12px;
    color: #333333;
}

QTableView::item {
    padding: 4px;
}

QTableView::item:selected {
    background-color: #70B9FE;
    color: #ffffff;
}

QHeaderView::section {
    background-color: #e6e6e6;
    border: none;
    padding: 6px;
    font-size: 13px;
    font-weight: bold;
    color: #333333;
}

QHeaderView::section:hover {
    background-color: #d6d6d6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 20pt; font-weight: bold; padding: 10px;</string>
     </property>
     <property name="text">
      <string>Payments</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUserID">
       <property name="text">
        <string>User ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxUserID"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelUserName">
       <property name="text">
        <string>User Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditUserName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelReservationID">
       <property name="text">
        <string>Reservation ID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditReservationID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSelectReservation">
       <property name="text">
        <string>Select Reservation:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxReservationID"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Amount:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditAmount"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelPaymentStatus">
       <property name="text">
        <string>Payment Status:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxPaymentStatus">
       <item>
        <property name="text">
         <string>Paid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pending</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelMethod">
       <property name="text">
        <string>Payment Method:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxPaymentMethod">
       <item>
        <property name="text">
         <string>Debit/Credit Card</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Online (Bank Transfer/Digital Wallet)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Payment Date:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="dateEditPaymentDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelTransactionID">
       <property name="text">
        <string>Transaction ID:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditTransactionID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelNotes">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QTextEdit" name="textEditPaymentNotes"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSearch">
     <property name="placeholderText">
      <string>Search by Reservation ID...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tablePayments">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>